core/src/test/java/org/elasticsearch/index/query/RandomQueryBuilder.java

```diff
@@ -47,6 +48,8 @@ public static QueryBuilder createQuery(Random r) {
             case 3:
                 return createMultiTermQuery(r);
             case 4:
+                return new GeoDistanceRangeQueryTests().createTestQueryBuilder();
```

Comment:
There's no reason for it. It looks to me like `GeoDIstanceRangeQueryTests.createTestQueryBuilder` and all referenced methods can be static? They don't require any instance state.